### PR TITLE
[gpsd] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
New: https://github.com/google/oss-fuzz/pull/8493

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
So upstream agreed?

File: projects/gpsd/project.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+homepage: "https://gpsd.io"
+language: c
+primary_contact: "gpsd-dev@lists.nongnu.org"
```
Comment (jonathanmetzman) [inline projects/gpsd/project.yaml:3]: This is not good. Members of the list will get emails but wont be able to access bugs.

Comment (0x34d) [reply projects/gpsd/project.yaml:3]: I understand I'm still trying to contact the GPSD team. 
If I get a member mail ID, who can fix bugs, I will change it. 
Until then, hold it.

**Comment by 0x34d** [comment]
> So upstream agreed?

I have sent an email. Let's see what happens. 
Afterward, I will try to contact them on GitLab issues.

**Comment by 0x34d** [comment]
Using Mr. Gary E. Miller as a backup.
If anything goes south.

**Comment by jonathanmetzman** [comment]
> Using Mr. Gary E. Miller as a backup. If anything goes south.

Can you please be more explicit. Does upstream agree to this?

**Comment by 0x34d** [comment]
> Can you please be more explicit. Does upstream agree to this?

Yes Mr. Gary E. Miller is the lead of GPSD.

**Comment by jonathanmetzman** [comment]
> > Can you please be more explicit. Does upstream agree to this?
> 
> Yes Mr. Gary E. Miller is the lead of GPSD.

I think I get it now but, i'll point out that youre still not being explicit.
Being explicit (which makes my work easier) would look something like this:
"Gary Miller is a member of the GPSD project and agrees to this integration"

**Review by jonathanmetzman — APPROVED** [review]
LGTM

Thanks for this integration!

**Pull request merged**